android/src/main/java/jp/manse/BrightcovePlayerView.java

```diff
@@ -725,4 +726,8 @@ public int hashCode() {
             return Objects.hash(height, width);
         }
     }
+    public void changeLiveIndicatorColor() {
+        Button liveResource = (Button) mediaController.getBrightcoveControlBar().findViewById(R.id.live);
+        liveResource.setTextColor(getResources().getColor(R.color.live_bmc_button));
```

Comment:
I don't see our colors.xml file updated. Is live_bmc_button already #FF8080?

Reply:
color file added.